ui/src/components/WorkerTable/index.jsx

```diff
@@ -95,6 +95,14 @@ export default class WorkerTable extends Component {
     });
   };
 
+  componentDidMount() {
+    const query = { sortBy: 'started', sortDirection: 'desc' };
+
+    this.props.history.replace({
```

Comment:
ah, wait, no :) this will always push new query. I think you forgot to add a check, if those values are not present in query, so you can set them. 

Ugh, maybe it's even better not to put those directly in url, since those are defaults, but just replace the default on line 111 - `{ sortBy: null, sortDirection: null }` => `{ sortBy: 'started', sortDirection: 'desc' }`
and if user changes sorting, it would only then change URL. 

Sorry :)

Reply:
So I had initially set the defaults like you mentioned here, but what that does is makes the user need to click on the `Started` column header two times to reverse the sort direction. First time they click, it'll add the query to the URL, then it'll switch on the second click. That's why I initially did it in the `render()` func. I guess I could just add the check in `componentDidMount()`??

Reply:
ah.. yeah.. then this, or you can also change the link that leads to this page, so user would end up here already with those values in query string

Reply:
I'm glad you guys understand this stuff! 🤣